tests/cypress/tests/standalone/navigation_test.ts

```diff
@@ -83,6 +83,13 @@ Object.values(BasicModes).forEach(mode => {
                 DashTable.getCell(2, 1).should('have.class', 'focused');
                 DashTable.getCell(3, 1).should('not.have.class', 'focused');
             });
+
+            it('can move out of the viewport in virtualized mode', () => {
+                DashTable.toggleScroll(true);
+                for (let i = 0; i < 25; i++) {
+                    DOM.focused.type(Key.ArrowDown);
+                }
+            });
```

Comment:
Nothing specific is being validated here. If the down arrow causes an error and/or does not update the selection the test will not fail.

Reply:
I tested this with the version of the table that had this error, and the test did end up failing, since there is a console error (which I thought would be implicitly included in the test failure conditions). Is there another way to catch console errors?

Reply:
Ah! You're right. Unhandled errors cause the tests to fail by default - did not realize that was the case.
https://docs.cypress.io/api/events/catalog-of-events.html#To-turn-off-all-uncaught-exception-handling

Reply:
Would still like the test do more validation. The implementation could change and the error be swallowed instead of being unhandled. The test would not fail in that case.